docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -67,15 +67,22 @@ It is however possible to use a custom configuration profile for your tests with
 This can be useful if you need for example to run some tests using a specific database which is not your default testing
 database.
 
+[role="primary asciidoc-tabs-sync-groovy"]
+.Groovy DSL
+****
 [source,groovy,subs=attributes+]
 ----
 test {
     systemProperty "quarkus.test.profile", "foo" <1>
 }
 ----
 
-or, if you use the Gradle Kotlin DSL:
+<1> The `foo` configuration profile will be used to run the tests.
```

Comment:
@gsmet I duplicate the `<1>` note on both tabs, otherwise it wasn't appearing. Is there a way to avoid duplicating that part?

Reply:
Yeah, that's how it should be done. The note is attached to the source block so you can't do it another way.